<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Banque</class>
 <widget class="QWidget" name="Banque">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>La Banque des SNIR</string>
  </property>
  <widget class="QGroupBox" name="groupBox_connexionBDD">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>611</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Connexion à la BDD</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>509</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_general">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_menu">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_menuGauche">
         <item>
          <widget class="QLabel" name="label_adresseIP">
           <property name="text">
            <string>Adresse IP :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_nomBDD">
           <property name="text">
            <string>Nom de la base de données :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_nomBDD_2">
           <property name="text">
            <string>Login :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_motDePasse">
           <property name="text">
            <string>Mot de passe :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_menudroite">
         <item>
          <widget class="QLineEdit" name="lineEdit_adresseIP">
           <property name="text">
            <string>172.18.58.8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nomBDD">
           <property name="text">
            <string>banque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_login">
           <property name="text">
            <string>snir</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="placeholderText">
            <string>snir</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_motDePasse">
           <property name="text">
            <string>snir</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="placeholderText">
            <string>snir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_connexionBDD">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Connexion à la base
de données</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_deconnexionBDD">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Déconnexion de la base
de données</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_infosBanque">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>611</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Informations de la banque</string>
   </property>
   <widget class="QPushButton" name="pushButton_afficherInfosComptes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Afficher les informations de tous les comptes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_infoBanque">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>571</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget_infoBanque">
       <column>
        <property name="text">
         <string>Numéro de compte</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nom</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Prénom</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Soldes en euros</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ajout">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_quitter">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
